frontend/lib/controllers/api_controller.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:dio/dio.dart';
+import 'package:frontend/sdk/user.dart';
+import 'package:frontend/sdk/workflow.dart';
+import 'package:get/get.dart';
+
+class ApiController extends GetxController {
+  static ApiController instance = Get.find();
+
+  User? user;
+  final Dio dio = Dio(
+    BaseOptions(
+      baseUrl: 'http://localhost:7001/',
```

Comment:
Please provide this from an env. variable

Reply:
fixed in [b77f227](https://github.com/Cleopha/ifttt-like/pull/84/commits/b77f22799e589ea1022e97242d2d048da4de21f1)